lib/deviantart_gallery_downloader.rb

```diff
@@ -115,9 +124,11 @@ def create_image_directories(folders)
       Dir.mkdir(@author_name) unless File.exists?(@author_name)
       Dir.chdir(@author_name) do
         folders.each do |folder|
-          `mv default-gallery Featured`if File.exists?('default-gallery') # For compatibility
           Dir.mkdir(folder[:name]) unless File.exists?(folder[:name])
         end
+        if File.exists?('default-gallery') # For compatibility
+          FileUtils.mv(Dir['default-gallery/*'],'Featured')
```

Comment:
I'm finding that replacing slash with backslash stops this from working (when the point of this change is that this command works when the bare `mv` command wasn't).  This is strange to me, since I understand that modern versions of Windows accept both kinds of slashes, but maybe Ruby internally works with only forward slash.  (All of the other path-construction in the code works fine with either kind of slash.)

Reply:
```ruby
module OS
  def OS.windows?
    (/cygwin|mswin|mingw|bccwin|wince|emx/ =~ RUBY_PLATFORM) != nil
  end

  if OS.windows?
    FileUtils.mv(Dir['default-gallery\*'],'Featured')
  else
    # the old way
  end

  # and so the other places containing slash
end
```
But as I said, many places need changes for this.